sw/device/lib/testing/usb_testutils_streams.c

```diff
@@ -476,10 +480,12 @@ status_t usb_testutils_streams_service(usb_testutils_streams_ctx_t *ctx) {
 }
 
 bool usb_testutils_streams_completed(const usb_testutils_streams_ctx_t *ctx) {
+  CHECK(ctx->nstreams <= UINT8_MAX, "ctx->nstreams must fit in a uint8_t");
```

Comment:
As this is a testutils, this function should be refactored to return a `status_t` wrapping the boolean (See the function `usb_testutils_simpleserial_send_byte` as an example.
```suggestion
  TRY_CHECK(ctx->nstreams <= UINT8_MAX, "ctx->nstreams must fit in a uint8_t");
```